src/api/app/policies/token_policy.rb

```diff
@@ -31,7 +33,10 @@ def update?
 
   def create?
     # TODO: when trigger_workflow is rolled out, remove the Flipper check
-    record.user == user && record.type != 'Token::Rss' && Flipper.enabled?(:trigger_workflow, user)
+    return false unless Flipper.enabled?(:trigger_workflow, user)
+    return false unless record.type != 'Token::Rss'
+
+    record.user == user || record.users_shared_among.include?(user) || record.groups_shared_among.map(&:users).flatten.include?(user)
```

Comment:
We need to fix this, we use this policy for any type of token. And for tokens different from workflow, `users_shared_among` and `groups_shared_among` are not defined.